lib/phraseapp_updater/phraseapp_api.rb

```diff
@@ -4,9 +4,10 @@
 
 class PhraseAppUpdater
   class PhraseAppAPI
-    def initialize(api_key, project_id)
+    def initialize(api_key, project_id, file_format)
```

Comment:
I think it would be better encapsulation if you passed the locale file class in instead of the file format.